src/pages/commits/BuildVariantAccordion.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Disclaimer } from "@leafygreen-ui/typography";
+
+const { gray } = uiColors;
+
+interface Props {
+  buildVariants: {
+    displayName: string;
+  }[];
+}
+export const BuildVariantAccordion: React.FC<Props> = ({ buildVariants }) => (
+  <ColumnContainer>
+    {buildVariants.map(({ displayName }) => (
+      <AccordionContainer key={displayName}>{displayName}</AccordionContainer>
+    ))}
+  </ColumnContainer>
+);
+
+const AccordionContainer = styled(Disclaimer)`
+  margin-top: 20px;
+  color: ${gray.dark2};
+  font-size: 14px;
+  width: 124px;
```

Comment:
Is this width needed?

Reply:
Grabbed this from the figma. I think it looks better when it is slightly skinnier than the commit chart column? So there is more white space. But not necessary at all, just a design choice.

Reply:
Here is the difference:
124px:
<img width="1425" alt="image" src="https://user-images.githubusercontent.com/55669681/128070752-a3afefe0-e3f8-49cd-8fdd-c4951e197dc2.png">

100%:
<img width="1424" alt="image" src="https://user-images.githubusercontent.com/55669681/128070770-b8f45673-f2b4-4815-97da-59f86450710c.png">